### PR TITLE
[CloudPC]Add the provisioningType property to CloudPC entity

### PR DESCRIPTION
**Pull request by xinliu22**
roadmap work item no: 15681

Add the `provisioningType` property to the cloudPC resource.

### CODE REVIEW

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]
@xinliu22 There's build error please update to make the build green

File: api-reference/beta/resources/cloudpc.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |statusDetails|[cloudPcStatusDetails](../resources/cloudpcstatusdetails.md)|The details of the Cloud PC status.|
 |userAccountType|[cloudPcUserAccountType](../resources/cloudpcorganizationsettings.md#cloudpcuseraccounttype-values)|The account type of the user on provisioned Cloud PCs. Possible values are: `standardUser`, `administrator`, and `unknownFutureValue`.|
 |userPrincipalName|String|The user principal name (UPN) of the user assigned to the Cloud PC.|
+|provisioningType|[cloudPcProvisioningType](../resources/cloudpcprovisioningpolicy.md#cloudpcprovisioningtype-values)|Specifies the type of licenses to be used when provisioning Cloud PCs using this policy. Possible values are dedicated and shared. It will be dedicated by default if it's not specified when creating provisioningPolicy. The possible values are: `dedicated`, `shared`, `unknownFutureValue`.|
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpc.md:70]:
```suggestion
|provisioningType|[cloudPcProvisioningType](../resources/cloudpcprovisioningpolicy.md#cloudpcprovisioningtype-values)|Specifies the type of licenses to be used when provisioning Cloud PCs using this policy. Possible values are: `dedicated`, `shared`, `unknownFutureValue`. Default value is dedicated if it's not specified when creating provisioningPolicy. |
```

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@xinliu22  Please create a roadmap workitem and attach the workitem number in the comment. Here is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/). If you don't have access, please do this firstly to gain access:
- Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
- Join the Microsoft Graph Roadmap Users ([graphroadmap@microsoft.com](mailto:graphroadmap@microsoft.com)) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

File: api-reference/beta/resources/cloudpc.md (hunk 1 above)
Comment (xinliu22) [reply api-reference/beta/resources/cloudpc.md]: Applied, thanks for your suggestion!

**Comment by xinliu22** [comment]
This PR depends on another [PR](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18537) that under review.

**Comment by RuiHou105** [comment]
@xinliu22  @ningjingbogithub  This PR is marked as draft and blocked. Could you please set it as Open and move forward?

**Comment by xinliu22** [comment]
@RuiHou105 We are waiting for this [PR](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18537) being merge, or the "microsoft-graph-docs apidoctor-beta" will always fail. Or is that okay lefting this PR in failure status?

**Comment by xinliu22** [comment]
And I'm wondering if we could combine these two PRs as single one.

**Comment by ningjingbogithub** [comment]
roadmap: https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/16483

**Review by RuiHou105 — APPROVED** [review]

**Comment by RuiHou105** [comment]
@Lauragra  @JarbasHorst  Hope you can help review this PR when you got time, thanks!

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Review by AshleyYangSZ — APPROVED** [review]

**Pull request merged**